src/nupic/algorithms/TemporalMemory.cpp

```diff
@@ -631,6 +632,10 @@ void TemporalMemory::compute(const SDR &activeColumns, bool learn,
                              const SDR &extraWinners)
 {
   activateDendrites(learn, extraActive, extraWinners);
+  compute(activeColumns, learn);
+}
+
+void TemporalMemory::compute(const SDR &activeColumns, bool learn) {
```

Comment:
This method does not call activateDendrites, but it should.  Try: `activateDendrites( learn );` the method has default arguments already, no need to make placeholder SDRs for the externalPredictiveInputs which are unused in this case.

Also change the above method to not call this one, so that activateDendrites isn't called twice.